manifest-api-preview.yml

```diff
@@ -3,14 +3,7 @@
 inherit: manifest-api-base.yml
 
 services:
-  - notify-aws
-  - notify-config
   - notify-db
```

Comment:
notify-db service is already in manifest-notify-base.yml. Do we still need it here?

Reply:
The lists aren't merged, so whenever we define `services` in a manifest we need to list all services again